servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H2ServerParentChannelInitializer.java

```diff
@@ -54,12 +52,10 @@ public void init(final Channel channel) {
                 config.headersSensitivityDetector();
         multiplexCodecBuilder.headerSensitivityDetector(headersSensitivityDetector::test);
 
-        final String frameLoggerName = config.frameLoggerName();
-        if (frameLoggerName != null) {
-            LogLevel logLevel = getNettyLogLevel(frameLoggerName);
-            if (logLevel != null) {
-                multiplexCodecBuilder.frameLogger(new Http2FrameLogger(logLevel, frameLoggerName));
-            }
+        final FixedLevelLogger frameLogger = config.frameLogger();
+        if (frameLogger != null) {
+            multiplexCodecBuilder.frameLogger(
+                    new ServiceTalkHttp2FrameLogger(frameLogger, config.frameLoggerUserData()));
```

Comment:
This code duplicates what we do in `H2ClientParentChannelInitializer`. Consider moving it to a utility method and reusing. It can be in one of these h2-initializers or in `ServiceTalkHttp2FrameLogger`.